manifests/source/export.pp

```diff
@@ -26,8 +32,10 @@
 # Copyright 2015 CoverMyMeds
 #
 define sentry::source::export (
-  $language = 'Other',
-  $env      = undef,
+  $language     = 'Other',
+  $organization = $::sentry::organization,
```

Comment:
If you set this to undef, your if statement in line 53 can be simpler. As is, do you even need the if statement in line 53?